docs/guides/admin/docs/modules/subtitles.md

```diff
@@ -0,0 +1,81 @@
+Subtitles
+=========
+
+This document records guidelines and conventions on how subtitles should be handled in Opencast.
+
+<div class=warn>
+These guidelines are new to Opencast 14. Not all parts of Opencast may yet adhere to them! 
```

Comment:
```suggestion
These guidelines are new to Opencast 15. Not all parts of Opencast may yet adhere to them! 
```

Reply:
Or maybe rewording to avoid update every time OC upgrades.

Reply:
Thanks for catching that again :see_no_evil:  
Maybe it's actually good having to look at this every major OC upgrade, so we're forced the re-evaluate if this warning is still appropriate?

Reply:
I don't know, It's straightforward easy to forget to update the tiny details in the docs. I just found in the dev docs some repos to Bitbucket that are from almost 5 years deprecated.

Reply:
Well, in that case the version number at least serves as an indicator that this part of the docs is likely outdated, and as a user you should proceed with care :D

Reply:
Fair Enough :)